client/css/style.css

```diff
@@ -107,6 +108,13 @@ body {
 	overflow: hidden; /* iOS Safari requires overflow rather than overflow-y */
 }
 
+@media screen and (orientation: portrait) {
+	/* Allow the main chat interface to merge into the background. */
+	body {
+		background: white;
```

Comment:
please remove that again.
I don't see why you need to set the color there for a padding issue.

People use dark themes you know and having some flash white is seldom a good idea :)

Reply:
This was to get the effect mentioned in my other comment, but definitely not necessary. Removed as requested.